cpp/src/comms/ipc/ipc.cpp

```diff
@@ -8,6 +8,9 @@
 #include <arrow/api.h>
 #include <arrow/io/api.h>
 #include <arrow/ipc/api.h>
+#include <arrow/ipc/json-integration.h>
```

Comment:
Did you accidentally autoformat this file while preparing this PR?

Reply:
I prefer not having this modification in here, but I'll push it through if you really think it is necessary. It'll be nice to get this PR merged because I think it may unblock the ffspec CI issues.

Reply:
@thomcom Apologies, this should have been WIP. @trxcllnt is reworking this entire file to remove the JSON usage and just use Arrow directly, but Arrow has changed its IPC messaging as of 0.14? where we unfortunately need to make changes to this file.